web/src/main/java/se/inera/intyg/rehabstod/service/sjukfall/GetActiveSickLeavesServiceImpl.java

```diff
@@ -102,10 +109,29 @@ private SickLeavesRequestDTO getRequest(
         request.setDoctorIds(filterRequest.getDoctorIds());
         request.setToSickLeaveLength(filterRequest.getToSickLeaveLength());
         request.setFromSickLeaveLength(filterRequest.getFromSickLeaveLength());
+        request.setDiagnosisChapters(convertDiagnosisChapters(filterRequest.getDiagnosisChapters()));
```

Comment:
I get a null point exception here if no diagnosis is selected in the filter request.

Reply:
Maybe we can send in a empty list if diagnosisChapters is null?

```java
filterRequest.getDiagnosisChapters() != null ? filterRequest.getDiagnosisChapters() : Collections.emptyList()
```

Reply:
I can do that but it will never be null from the frontend, it's just because we updated the name after I had merged the frontend PR.